packages/debug/src/browser/debug-configuration-manager.ts

```diff
@@ -254,21 +274,28 @@ export class DebugConfigurationManager {
 
     protected async doOpen(model: DebugConfigurationModel): Promise<EditorWidget> {
         let uri = model.uri;
-        if (!uri) {
+        const configFileUri = this.getFolderConfigurationUri(model);
+        const fileStat = await this.filesystem.getFileStat(configFileUri.toString());
```

Comment:
This line throws an error when the the parent folder of the file does not exist. Maybe `exists` does the trick? But the documentation is really not very precise on this.